lib/util/exec/index.spec.ts

```diff
@@ -12,7 +10,7 @@ import { exec } from '.';
 
 const cpExec: jest.Mock<typeof _cpExec> = _cpExec as any;
 
-jest.mock('child_process');
+jest.mock('./common');
```

Comment:
wasn't this already changed on other refactor PR? 🤔

Reply:
nope, I missed that one. 
it's less obvious change. nonetheless, it could have be done in the previous PR.